### PR TITLE
Added issue description and workaround for installer crash problem.

### PR DESCRIPTION
**Pull request by masteryyh**
**Related Issue:**
[#2937](https://github.com/harvester/harvester/issues/2937)

### CODE REVIEW

File: docs/install/iso-install.md [hunk 1]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
 
-:::note
+<!-- :::note
 In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation. See issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.
-:::
+::: -->
+
+## Known Issue
+
+### Installer may crash when using an older graphics card/monitor
```
Comment (vickyhella) [inline docs/install/iso-install.md:62]:
```suggestion
### Installer may crash when using an older graphic card/monitor
```

File: docs/install/iso-install.md [hunk 2]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
+In some cases, if you are using an older graphics card/monitor, you may encounter an `panic: invalid dimensions` error with ISO installation.
```
Comment (vickyhella) [inline docs/install/iso-install.md:64]:
```suggestion
In some cases, if you are using an older graphic card/monitor, you may encounter a `panic: invalid dimensions` error during ISO installation.
```

File: docs/install/iso-install.md [hunk 3]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
+![invalid-dimensions.png](/img/v1.1/install/invalid-dimensions.png)
+
+This is a known issue we are working on, and will be fixed in the future release. Here is a workaround for this issue:
+1. Boot up with the ISO, and press `E` to edit first menu entry;
```
Comment (vickyhella) [inline docs/install/iso-install.md:69]:
```suggestion
1. Boot up with the ISO, and press `E` to edit the first menu entry:
```

File: docs/install/iso-install.md [hunk 4]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
+## Known Issue
+
+### Installer may crash when using an older graphics card/monitor
+
+In some cases, if you are using an older graphics card/monitor, you may encounter an `panic: invalid dimensions` error with ISO installation.
+
+![invalid-dimensions.png](/img/v1.1/install/invalid-dimensions.png)
+
+This is a known issue we are working on, and will be fixed in the future release. Here is a workaround for this issue:
```
Comment (vickyhella) [inline docs/install/iso-install.md:68]:
```suggestion
This is a known issue we are working on, and will be fixed in future releases. Here is a workaround for this issue:
```

File: docs/install/iso-install.md [hunk 5]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+1. Boot up with the ISO, and press `E` to edit first menu entry;
+   ![grub-menu.png](/img/v1.1/install/grub-menu.png)
+2. Append `vga=792` to the line started with `$linux`;
```
Comment (vickyhella) [inline docs/install/iso-install.md:71]:
```suggestion
2. Append `vga=792` to the line started with `$linux`:
```

File: docs/install/iso-install.md [hunk 6]
```diff
@@ -52,6 +53,21 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+   ![edit-menu-entry.png](/img/v1.1/install/edit-menu-entry.png)
+3. Press `Ctrl+X` or `F10` to boot up;
```
Comment (vickyhella) [inline docs/install/iso-install.md:73]:
```suggestion
3. Press `Ctrl+X` or `F10` to boot up.
```

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Review by vickyhella — APPROVED** [review]
Just a few nits. The rest LGTM and thanks!

**Review by vickyhella — APPROVED** [review]

**Review by guangbochen — APPROVED** [review]

**Pull request merged**